BUILD

```diff
@@ -14,30 +14,4 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-
-exports_files(["deployment.properties"], visibility = ["//visibility:public"])
-
-# When a Bazel build or test is executed with RBE, it will be executed using the following platform.
-# The platform is based on the standard rbe_ubuntu1604 from @bazel_toolchains,
-# but with an additional setting dockerNetwork = standard because our tests need network access
-platform(
-    name = "rbe-ubuntu1604-network-standard",
-    parents = ["@bazel_toolchains//configs/ubuntu16_04_clang/11.0.0/bazel_3.0.0/config:platform"],
-    remote_execution_properties = """
-        properties: {
-            name: "container-image"
-            value: "docker://gcr.io/grakn-dev/rbe_platform@sha256:435403f84a20db7ac779bf2688dce6bd6463d62608c0731cfb944cd0b74b35d8"
-        }
-        properties: {
-          name: "dockerNetwork"
-          value: "standard"
-        }
-        """,
-)
-
-py_library(
-    name = "common",
-    srcs = ["common.py"],
-    visibility = ["//visibility:public"]
-)
+#
```

Comment:
new line

Reply:
Solved in https://github.com/graknlabs/dependencies/pull/136/commits/e44a270ac97b4d28e56e409d91fd75663ccb164c